docs/source/library/EccFramework.rst

```diff
@@ -0,0 +1,7 @@
+apply_ecc
+========================
+
+This function takes a quantum circuit either as a file name of a openQasm quantum circuit or as a Qiskit QuantumCircuit
```

Comment:
Put the `QuantumCircuit` either in a :code: environment or also in a sphinx autoref environment.